//in 5.6.2

#include "../../std_lib_facilities.h"

int main()
try {
	vector<int> v;
	for (int x; cin >> x;) {
		v.push_back(x);
	}
	for (int i = 0; i <= v.size(); ++i) {
		cout << "v[" << i << "] == " << v[i] << '\n';
	}
}catch (out_of_range) {
	cerr << "Ops! Range error\n";

	keep_window_open();       //i don't know why,but this keep_window_open(); didn't work.
	int y;                    //so I added this.
	cin >> y;
	return 1;
}catch (...) {
	cerr << "Exception: something went wrong\n";
	
	keep_window_open();
	return 2;
}
